src/casus/blocks/PrintBlock.js

```diff
@@ -0,0 +1,19 @@
+//@flow strict
+
+import UnaryOperationBlock from './UnaryOperationBlock.js';
+import type {DataType} from './DataType.js';
+
+class PrintBlock extends UnaryOperationBlock {
+
+	constructor(paramType: DataType) {
+		super(paramType, 'VOID', 'print');
+	}
+
+	evaluate(): null {
+		const res=this.rChild.evaluate();
+		console.log(res);
```

Comment:
Will this print somewhere else at production? I thought we weren't supposed to use the console.log function.

Reply:
In general, yeah we shouldn't log in production, but for this case we don't have any other way to print variables since we don't have a display up for the game yet, so this is the best functionality the code can have.

This isn't debug code though, this is (for now at least) the function that the print block is suppose to perform.

Reply:
Sounds good.